### ISSUE TITLE
Support for LaTeX math environments like in Jupyter notebooks?

### ISSUE THREAD

**Issue by mgeier** [executablebooks/MyST-Parser#380]
I was trying to use MyST-Parser with `nbsphinx` in https://github.com/spatialaudio/nbsphinx/pull/560, but it doesn't seem to support math environments like

```
\begin{equation*}
a
\end{equation*}
```

**Describe the solution you'd like**

It would be great if that worked just like in Jupyter notebooks.

Is there an extension I can enable to get this?

**Comment by chrisjsewell** [comment]
> Is there an extension I can enable to get this?

See https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#direct-latex-math,
which uses the `amsmath_plugin`: https://mdit-py-plugins.readthedocs.io/en/latest/#math

**Comment by mgeier** [comment]
Cool, that seems to be it!

Is there by any chance a Python package that has the same extensions enabled that are also used in the Jupyter project?

If not, I guess I'll have to create my own ...

**Comment by chrisjsewell** [comment]
> Is there by any chance a Python package that has the same extensions enabled that are also used in the Jupyter project?

I'm not sure I understand what you mean by this. What is the goal you want to achieve with the python package?

**Comment by chrisjsewell** [comment]
> And with "easy" I mean: one Python package, one entry in extensions in conf.py, nothing else.

So basically myst-parser with hard-coded configuration 🤷

**Comment by mgeier** [comment]
Exactly. Sorry that I wasn't clearer about that.

**Comment by chrisjsewell** [comment]
I mean obviously it is entirely up to you if you want to re-write myst-parser in your own package.
But it would seem better to me, to break up myst-parser into one "simpler" core package, and then one (or more) packages that use it as the base implementation to extend on (including myst-parser itself).
This is a similar to #347

**Comment by choldgraf** [comment]
> But it would seem better to me, to break up myst-parser into one "simpler" core package, and then one (or more) packages that use it as the base implementation to extend on (including myst-parser itself).

Yep - that's what I imagined too re: "distributions". I think that we should be careful that we don't dilute the meaning of "MyST" in the process (e.g., we may want to consider that a distribution of MyST must be a strict superset of MyST syntax, it cannot _disable_ any core MyST syntax) but it'd be a useful extension point to be able to have `myst`, `myst-jupyter`, etc.

**Comment by chrisjsewell** [comment]
ok well I will close this anyhow 👍

**Comment by mgeier** [comment]
I just had an epiphany: `nbsphinx` already has Markdown support via Jupytext! I should have thought about this earlier, it's actually quite obvious.

Everything already works as expected, I just had to document it: https://github.com/spatialaudio/nbsphinx/pull/561

So I won't have to create my own MyST-based Python package after all!

I will not need anything that we just discussed in this issue, but it might still be interesting for other people who might want to use MyST-Parser with Jupyter-like Markdown extensions.

But I'm fine with closing this, I won't be needing it anymore.

Thanks for your help, though!
